ts/IterableInbox.tsx

```diff
@@ -131,6 +134,9 @@ const IterableInbox = ({
       setRowViewModels(newRowViewModels)
       inboxDataModel.setMessageAsRead(id)
       setSelectedRowViewModelIdx(index)
+
+      Iterable.trackInAppOpen(rowViewModels[index].inAppMessage, IterableInAppLocation.inbox)
```

Comment:
Can we put this into `IterableInboxMessageDisplay.tsx`? All the data is there, it'd be more fitting to put it into the component mount of it.

Reply:
Ah I see, okay!